code/src/java/pcgen/gui2/kits/KitPanel.java

```diff
@@ -283,12 +283,14 @@ public KitTreeViewModel(CharacterFacade character, boolean isAvailModel)
 				ListFacade<KitFacade> kitList = new DefaultListFacade<KitFacade>(character.getAvailableKits());
 				kits.setDelegate(kitList);
 				character.getKits().addListListener(this);
-				columns = Arrays.asList(new DefaultDataViewColumn("in_source", String.class, false)); //$NON-NLS-1$
+				columns = Arrays.asList(new DefaultDataViewColumn("in_descrip", String.class), //$NON-NLS-1$
```

Comment:
Well, imo the table column would be more suited towards short descriptions whereas the info pane is suited to long descriptions. But since we don't have anyway to distinguish long and short descriptions they are one and the same at the moment.